aggregator_test.go

```diff
@@ -70,6 +70,10 @@ func TestCreateStorage(t *testing.T) {
 }
 
 func TestStartService(t *testing.T) {
+	// It is necessary to perform migrations for this test
+	// because the service won't run on top of an empty DB.
+	*main.AutoMigratePtr = true
```

Comment:
Maybe something like `EnableAutoMigrate()` and `defer DisableAutoMigrate()` would be better, but it's definitely a nitpick :)

Reply:
@Sergey1011010 I wanted to avoid adding too much code for this because I don't think it should be used from too many places.